lightning/src/ln/channelmanager.rs

```diff
@@ -6164,7 +6202,7 @@ where
 						},
 					}
 				},
-				Some(ChannelPhase::Funded(_)) | Some(ChannelPhase::UnfundedOutboundV1(_)) => {
+				Some(_) => {
```

Comment:
Why is this change needed? Aren't the match branch exhaustive, both when dual_funding option is set and when not?
The wildcard branch like this can mask some problems in the future.